src/pages/workflows/workflow/common/WorkflowModal.tsx

```diff
@@ -20,52 +22,104 @@ interface NamespaceNameSectionProps {
   name: string | undefined;
   setWorkflowNamespace: (value: string) => void;
   setWorkflowName: (value: string) => void;
+  errors: any;
 }
 
 interface SynopsisSnapshotSectionProps {
   synopsis: string;
   setWorkflowSynopsis: (value: string) => void;
+  errors: any;
 }
 
+const constraints = {
+  namespace: {
+    length: { maximum: 254 },
```

Comment:
Does Broad Methods Repo allow namespace to be 254 characters?! 😱

Reply:
Just did a test and it seems like BMR allows up to a total of 250 characters for the namespace/name. I wonder if it's possible to introduce logic into the value for length, or if we should just set a limit of 100 (or 125) for namespace and name?

Reply:
We chatted about this a little in mob time. I think the best option might be to keep track of the total length of name + namespace using state, and show an error + disable the button while that length is too long.